config/fusion_reactor.yml

```diff
@@ -0,0 +1,19 @@
+# Cloud Foundry Java Buildpack
+# Copyright 2017 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Configuration for the FusionReactor framework
+---
+version: 1.+
+repository_root: "{default.repository.root}/fusionreactor"
```

Comment:
This will need to be a repository hosted by Fusion Reactor team.  The repository structure is [documented](https://github.com/cloudfoundry/java-buildpack/blob/master/docs/extending-repositories.md) and a number of our external integrations provide [great](https://github.com/cloudfoundry/java-buildpack/blob/master/config/jrebel_agent.yml#L19) [examples](https://dl.zeroturnaround.com/jrebel/index.yml).  [Let me know](https://cloudfoundry.slack.com/messages/C03F5ELTK) if you have trouble understanding what's expected here and I can provide help.